Sdext.tex

```diff
@@ -36,7 +36,7 @@ \section{Debug Mode} \label{debugmode}
 \item If \FcsrDcsrStoptime is 0 then \Rtime continues to update. If
     it is 1 then \Rtime will not update. It will resynchronize with
     \Rmtime after leaving Debug Mode.
-\item The {\tt wfi} instruction acts as a {\tt nop}.
+\item The {\tt wfi}, {\tt wrs.sto}, {\tt wrs.nto} instructions act as a {\tt nop}.
```

Comment:
```suggestion
\item Instructions that place the hart into a stalled state act as a {\tt nop}.
    This includes {\tt wfi}, {\tt wrs.sto}, and {\tt wrs.nto}).
```

Reply:
Updated in this commit:
https://github.com/riscv/riscv-debug-spec/pull/838/commits/7cddfb3769a47bd0c9ea6158749832e3bfd75e78